apps/paramaps/itkimage2paramap.xml

```diff
@@ -2,46 +2,48 @@
 <executable>
   <category>Informatics.Converters</category>
   <title>Encode DICOM Parametric Map</title>
-  <description>Convert a parametric map provided in any of the formats supported by ITK, such as NRRD or NIFTI, as a DICOM Parametric Map image object.</description>
+  <description>Convert a parametric map provided in any of the formats supported by ITK, such as NRRD or NIFTI, into a DICOM Parametric Map image object.</description>
   <version>1.0</version>
   <documentation-url>https://github.com/QIICR/dcmqi</documentation-url>
   <license></license>
   <contributor>Andrey Fedorov(BWH), Christian Herz(BWH)</contributor>
   <acknowledgements>This work is supported in part the National Institutes of Health, National Cancer Institute, Informatics Technology for Cancer Research (ITCR) program, grant Quantitative Image Informatics for Cancer Research (QIICR) (U24 CA180918, PIs Kikinis and Fedorov).</acknowledgements>
 
   <parameters>
+
+    <file>
+      <name>metaDataFileName</name>
+      <label>JSON metadata file</label>
+      <channel>input</channel>
+      <index>0</index>
+      <description>File name of the JSON files containing metadata attributes.</description>
+    </file>
+
```

Comment:
I would position the metadata file on index 1 and the parametric file name on index 0 since the metadata file describes data coming from the parametric map file